test/spec/validations/crossing_ways.js

```diff
@@ -383,6 +388,11 @@ describe('iD.validations.crossing_ways', function () {
         verifySingleCrossingIssue(validate(), null);
     });
 
+    it('flags road tunnel crossing road on the same layer', function() {
+        createWaysWithOneCrossingPoint({ highway: 'residential', tunnel: 'yes', layer: '-1' }, { highway: 'tertiary', layer: '-1' });
+        verifySingleCrossingIssue(validate(), {});
+    });
+
```

Comment:
```suggestion

    it('flags a crossing where only one feature is indoor', () => {
        createWaysWithOneCrossingPoint({ highway: 'path' }, { highway: 'path', level: '1' });
        verifySingleCrossingIssue(validate(), {});
    });
```
i think one more test case would be good, to confirm that this behaviour is unchanged (1 indoor feature + 1 non-indoor feature)

Reply:
thanks for the suggestion, added as-is